inference-engine/src/gna_plugin/transformations/broadcast_const.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <openvino/cc/ngraph/itt.hpp>
+
+#include "transformations/broadcast_const.hpp"
+
+#include "transformations/utils/transformation_helper.hpp"
+
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include "legacy/ngraph_ops/eltwise.hpp"
+#include <transformations/utils/utils.hpp>
+
+#include <vector>
+#include <functional>
+#include <numeric>
+
+using namespace GNAPluginNS;
+
+NGRAPH_RTTI_DEFINITION(BroadcastAddMultiplyConst, "BroadcastAddMultiplyConst", 0);
+
+using Node = std::shared_ptr<ngraph::Node>;
+using Nodes = std::vector<Node>;
+using Input = ngraph::Input<ngraph::Node>;
+using Inputs = std::vector<Input>;
+using AxisSet = std::set<size_t>;
+
+namespace {
+
+bool HasDynamicShape(Node node) {
+    const auto & shape = node->get_output_partial_shape(0);
+    return shape.is_dynamic();
+}
+
+bool DoTransformation(Node const_node, Node eltwise_node) {
+    if (HasDynamicShape(const_node) || HasDynamicShape(eltwise_node))
+        return false;
+    const ngraph::Shape & eltwise_out_shape = eltwise_node->get_output_tensor(0).get_shape();
+
+    auto broadcast_const = ngraph::opset8::Constant::create(ngraph::element::Type_t::i64,
+                                         ngraph::Shape{eltwise_out_shape.size()}, eltwise_out_shape);
+
+    auto new_const_node = ngraph::op::util::make_try_fold<ngraph::opset8::Broadcast>(const_node,
+                                                                                     broadcast_const,
+                                                                                     ov::op::BroadcastType::NUMPY);
```

Comment:
Also, it would be nice to check broadcast type parameter of Add/Multiply itself.